templates/assets/images/ni-powered-by-logo.svg

```diff
@@ -0,0 +1,15 @@
+<svg class="ons-svg-logo ons-svg-logo--nisra" xmlns="http://www.w3.org/2000/svg" width="170" height="56" viewBox="0 1 170 54" aria-labelledby="nisra-logo-en-alt">
```

Comment:
If you want to avoid using two files you can do: 
```python
powered_by_logo: str = read_file("./templates/assets/images/ni-logo.svg").replace("#3d7cc9", "#222").replace("#cddb00", "#222").replace("#00205c", "#222")
```